zipline/pipeline/loaders/blaze.py

```diff
@@ -648,11 +648,31 @@ def overwrite_from_dates(asof, dense_dates, sparse_dates, asset_idx, value):
     -------
     overwrite : Float64Overwrite
         The overwrite that will apply the new value to the data.
+
+    Notes
+    -----
+    The overwrite is forward filled in the sense that it applies from the index
+    of the ``asof`` in the ``dense_dates`` to the index of the next date in the
```

Comment:
I would have called the `requested_dates` and `loaded_dates`.  There's no
guarantee that the data from a dataset will be more or less dense than the
requested data.

On Thu, Nov 5, 2015 at 3:55 PM, Joe Jevnik <notifications@github.com> wrote:

> In zipline/pipeline/loaders/blaze.py
> <https://github.com/quantopian/zipline/pull/808#discussion_r44067830>:
>
> > @@ -648,11 +648,31 @@ def overwrite_from_dates(asof, dense_dates, sparse_dates, asset_idx, value):
> >      -------
> >      overwrite : Float64Overwrite
> >          The overwrite that will apply the new value to the data.
> > +
> > +    Notes
> > +    -----
> > +    The overwrite is forward filled in the sense that it applies from the index
> > +    of the ``asof`` in the ``dense_dates`` to the index of the next date in the
>
> I think they are reasonable, what do you suggest instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/zipline/pull/808/files#r44067830>.
>